pkg/metadataapi/server/middleware/auth.go

```diff
@@ -188,7 +188,9 @@ func (ka *KubernetesAuthenticator) injector(mgrs *builder.MetadataBuilder, tags
 			logContext = fmt.Sprintf("tenants/%s/runs/%s/steps/%s", claims.RelayTenantID, claims.RelayRunID, claims.RelayName)
 		}
 
-		if ka.logServiceClient != nil && logContext != "" {
+		// HACK Temporarily disable the logging of steps, pending the next phase of logging improvements.
+		if action.Type().Singular == model.ActionTypeTrigger.Singular &&
```

Comment:
FWIW, since these aren't pointers, I think you can just do `action.Type() == model.ActionTypeTrigger`.